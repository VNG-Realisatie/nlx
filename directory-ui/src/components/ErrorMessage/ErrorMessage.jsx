// Copyright Â© VNG Realisatie 2018
// Licensed under the EUPL

import React from 'react'
import { StyledErrorMessage } from './ErrorMessage.styles'

const ErrorMessage = () => (
    <StyledErrorMessage>
        <h1>Failed to load information</h1>
        <p>
            Requested information is not available.
            <br />
            We apologize for any inconvenience.
        </p>
    </StyledErrorMessage>
)

export default ErrorMessage
