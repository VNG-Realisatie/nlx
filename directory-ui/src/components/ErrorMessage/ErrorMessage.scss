@import "~bourbon/core/bourbon";

.ErrorMessage {
  transform: translate(-50%,-50%);
  @include position(absolute, 50% null null 50%);
}
