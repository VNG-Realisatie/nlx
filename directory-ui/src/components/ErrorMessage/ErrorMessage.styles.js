import styled from 'styled-components'

export const StyledErrorMessage = styled.div`
    transform: translate(-50%, -50%);
    top: 50%;
    left: 50%;
    position: absolute;
`
