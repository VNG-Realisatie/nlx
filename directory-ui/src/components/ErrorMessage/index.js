// Copyright Â© VNG Realisatie 2018
// Licensed under the EUPL
//

import React from 'react'
import { Container } from './index.styles'

const ErrorMessage = () => (
  <Container>
    <h1>Kon geen informatie ophalen</h1>
    <p>
      Probeer het later nog eens.
      <br />
      Excuus voor het ongemak.
    </p>
  </Container>
)

export default ErrorMessage
