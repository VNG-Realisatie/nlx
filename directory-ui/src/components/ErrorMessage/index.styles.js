// Copyright Â© VNG Realisatie 2020
// Licensed under the EUPL
//

import styled from 'styled-components'

export const Container = styled.div`
  transform: translate(-50%, -50%);
  top: 50%;
  left: 50%;
  position: absolute;
`
