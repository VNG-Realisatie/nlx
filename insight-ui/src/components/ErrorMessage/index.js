// Copyright Â© VNG Realisatie 2020
// Licensed under the EUPL
//

import React from 'react'
import { node, string } from 'prop-types'
import { StyledCard } from './index.styles'

const ErrorMessage = ({ title, children, ...props }) => (
  <StyledCard {...props}>
    <h1>{title}</h1>
    {children}
  </StyledCard>
)

ErrorMessage.propTypes = {
  title: string.isRequired,
  children: node,
}

export default ErrorMessage
