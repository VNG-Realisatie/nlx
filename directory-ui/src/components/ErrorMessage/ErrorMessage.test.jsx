import React from 'react'
import { shallow } from 'enzyme'

import ErrorMessage from './ErrorMessage'

it('should match the snapshot', () => {
    expect(shallow(<ErrorMessage />)).toMatchSnapshot()
})
