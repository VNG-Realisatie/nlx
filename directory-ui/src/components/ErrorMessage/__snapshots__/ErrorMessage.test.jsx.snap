// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<styled.div>
  <h1>
    Failed to load information
  </h1>
  <p>
    Requested information is not available.
    <br />
    We apologize for any inconvenience.
  </p>
</styled.div>
`;
